docs/Walk_through.rst

```diff
@@ -62,12 +62,12 @@ start.py
             simulation.declare_round_endowment(resource='labor_endowment', units=1, product='labor')
             simulation.declare_perishable(good='labor')
 
-            simulation.panel('household', possessions=['money', 'GOOD'],
+            simulation.panel('Household', possessions=['money', 'GOOD'],
```

Comment:
There are 3 ways to fetch an agent:
1. From a Python variable by its index
2. From a global hash table by string
3. From a global hash table by id

In this case, `simulation.panel` should have fetched the agents from the Python variable directly instead of a string / from a global hash table.